epevermodbus/command_line.py

```diff
@@ -56,112 +57,178 @@ def main():
+        output["total_consumed_energy"] = controller.get_total_consumed_energy()
+        output["generated_energy_today"] = controller.get_generated_energy_today()
+        output["generated_energy_this_month"] = controller.get_generated_energy_this_month()
+        output["generated_energy_this_year"] = controller.get_generated_energy_this_year()
+        output["total_generated_energy"] = controller.get_total_generated_energy()
+        output["current_device_time"] = controller.get_rtc()
+        
+        output["rated_charging_current"] = controller.get_rated_charging_current()
+        output["rated_load_current"] = controller.get_rated_load_current()
+        output["battery_real_rated_voltage"] = controller.get_battery_real_rated_voltage())
+        output["battery_type"] = controller.get_battery_type()
+        output["battery_capacity"] = controller.get_battery_capacity()
+        output["temperature_compensation_coefficient"] = controller.get_temperature_compensation_coefficient()
+        output["battery_voltage_control_register_names"] = controller.battery_voltage_control_register_names)
+        output["battery_voltage_control_registers"] = controller.get_battery_voltage_control_registers()
+        output["over_voltage_disconnect_voltage"] = controller.get_over_voltage_disconnect_voltage()
+        output["charging_limit_voltage"] = controller.get_charging_limit_voltage())
+        output["over_voltage_reconnect_voltage"] = controller.get_over_voltage_reconnect_voltage()
+        output["equalize_charging_voltage"] = controller.get_equalize_charging_voltage())
+        output["boost_charging_voltage"] = controller.get_boost_charging_voltage())
+        output["float_charging_voltage"] = controller.get_float_charging_voltage())
+        output["boost_reconnect_charging_voltage"] = controller.get_boost_reconnect_charging_voltage()
+        output["low_voltage_reconnect_voltage"] = controller.get_low_voltage_reconnect_voltage()
+        output["under_voltage_recover_voltage"] = controller.get_under_voltage_recover_voltage()
+        output["under_voltage_warning_voltage"] = controller.get_under_voltage_warning_voltage()
+        output["low_voltage_disconnect_voltage"] = controller.get_low_voltage_disconnect_voltage()
+        output["discharging_limit_voltage"] = controller.get_discharging_limit_voltage())
+        output["battery_rated_voltage"] = controller.get_battery_rated_voltage()
+        output["default_load_on_off_in_manual_mode"] = controller.get_default_load_on_off_in_manual_mode()
+        output["equalize_duration"] = controller.get_equalize_duration()
+        output["boost_duration"] = controller.get_boost_duration()
+        output["battery_discharge"] = controller.get_battery_discharge()
+        output["battery_charge"] = controller.get_battery_charge()
+        output["charging_mode"] = controller.get_charging_mode()
+        print(output)
```

Comment:
Please use the builtin ``json.dumps(output)`` here.
Using a library we make sure that the json will always be well-formatted and can be read across platforms.
The Python ``str()`` output of a dict should not be considered stable.